### ISSUE TITLE
Partially minified output

### ISSUE THREAD

**Issue by kombucha** [VFK/gulp-html-replace#28]
Hi,

I've encountered an issue with this plugin when piping some "partially" minified.
I'm using jade templating with non-prettified output. The html output is still more than one line because I have some inline javascript that is outputted as is on multiple line (analytics code).
For that reason, we don't get in the [special case](https://github.com/VFK/gulp-html-replace/blob/master/lib/parser.js#L25) where you split the lines manually and as a result, the output is mangled and incomplete. 
I have removed the linefeed from the google analytics snippet as a workaround for now.

Cheers

**Comment by VFK** [comment]
Hi @kombucha can you show me this partially minified code please?

**Comment by kombucha** [comment]
Hi,

It looks something like this

```html
<!DOCTYPE html><html lang="fr" class="not-ie no-js"><head><title>Some</title><meta charset="UTF-8"><meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no"><link rel="stylesheet" href="styles/main.css"><link rel="icon" href="assets/images/favicons/favicon.ico" sizes="16x16 32x32 48x48 64x64" type="image/x-icon"><link rel="apple-touch-icon" href="assets/images/favicons/apple-touch-icon.png"><script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
ga('create', '__MY_API_KEY__', 'auto');
ga('send', 'pageview');</script></head><body><div>Some content</div><!-- build:js--><script src="bower_components/skrollr/src/skrollr.js"></script><script src="bower_components/skrollr-menu/src/skrollr.menu.js"></script><script src="bower_components/cheet.js/cheet.js"></script><script src="scripts/utils.js"></script><script src="scripts/nav.js"></script><script src="scripts/easter-egg.js"></script><script src="scripts/main.js"></script><!-- endbuild--></body></html>
```
And if I pipe this into htmlReplace : 

```js
gulp.task('test', function () {
    gulp.src('src/index-test.html')
        .pipe(htmlReplace({
            js: 'scripts/' + config.scriptName
        }))
        .pipe(gulp.dest(config.dist));
});
```

I get this : 

```html
<!DOCTYPE html><html lang="fr" class="not-ie no-js"><head><title>Some</title><meta charset="UTF-8"><meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no"><link rel="stylesheet" href="styles/main.css"><link rel="icon" href="assets/images/favicons/favicon.ico" sizes="16x16 32x32 48x48 64x64" type="image/x-icon"><link rel="apple-touch-icon" href="assets/images/favicons/apple-touch-icon.png"><script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
ga('create', '__MY_API_KEY__', 'auto');
```

**Comment by VFK** [comment]
Ok, It probably took much more time than it should, but i just released a new version - 1.5.0
I've rewritten a parsing engine so the plugin doesn't care about how your HTML look anymore, all it needs is build tags. @kombucha if you're still using this plugin then please test the new version, for me it works with your code just fine.

**Comment by VFK** [comment]
So, the latest version should fix this issue, i tested it many times. I'm closing this issue because of no feedback (but it's totally ok :) ), but feel free to open a new one if the latest version doesn't work for you.
